scripts/deploy.ts

```diff
@@ -98,7 +98,8 @@ async function main() {
       'phase2-assets/assets/deploy_crv.ts',
       'phase2-assets/assets/deploy_cvx.ts',
       'phase2-assets/collaterals/deploy_pyusd.ts',
-      'phase2-assets/collaterals/deploy_sky_susds.ts'
+      'phase2-assets/collaterals/deploy_sky_susds.ts',
+      'deploy_origin_oeth.ts',
```

Comment:
use full path `phase2-assets/collaterals/deploy_origin_oeth.ts`

Reply:
Done in this commit [eaa8b4e](https://github.com/reserve-protocol/protocol/pull/1239/commits/eaa8b4e622294558811e5fd942def655d0d31f54)